spec/services/prison_offender_manager_service_spec.rb

```diff
@@ -28,14 +30,14 @@
     describe '#get_pom_at' do
       it "can fetch a single POM for a prison",
          vcr: { cassette_name: 'prison_api/pom_service_get_pom_ok' } do
-        pom = described_class.get_pom_at('LEI', staff_id)
+        pom = prison.get_pom_at(staff_id)
         expect(pom).not_to be nil
       end
 
       it "raises an exception when fetching a pom if they are not a POM",
          vcr: { cassette_name: 'prison_api/pom_service_get_pom_none' } do
         expect {
-          described_class.get_pom_at(prison.code, 1234)
+          prison.get_pom_at(1234)
```

Comment:
This behaviour doesn't feel very nice. It's uncatchable and slightly awkward. Maybe we should mirror ActiveRecords interface so that we can decide whether to throw or not?